pkg/plugin/aggregation/status_test.go

```diff
@@ -25,14 +25,14 @@ func TestUpdateStatus(t *testing.T) {
 		expectedStatus string
 	}{
 		{
-			name:           "empty is complete",
+			name:           "empty is post-processing",
 			pluginStatuses: []string{},
-			expectedStatus: "complete",
+			expectedStatus: "post-processing",
```

Comment:
I did have to modify a few tests, but it genuinely seems to me like the model was just _slightly_ broken here so they really do need updated. No one should assume that anything is available for download or that sonobuoy is completely done while in this post-processing phase.